### PR TITLE
NXP-21644: refactor nuxeo-search-form

### PR DESCRIPTION
**Pull request by nelsonsilva**
TBR but could use some testing

### CODE REVIEW

**Comment by adilioaraujo** [comment]
:+1:

File: index.html [hunk 1]
```diff
@@ -96,7 +96,7 @@
 <body unresolved>
   <span id="browser-sync-binding"></span>
 
-  <nuxeo-connection url="/nuxeo"></nuxeo-connection>
+  <nuxeo-connection url="http://webui-demo.nuxeo.com/nuxeo"></nuxeo-connection>
```
Comment (guirenard) [inline index.html]: Keep it working for dev mode ;)

**Comment by guirenard** [comment]
:+1:

File: index.html (hunk 1 above)
Comment (nelsonsilva) [inline index.html]: thx ;)

**Pull request closed without merging**